tests/e2e/e2e_test.go

```diff
@@ -1,135 +1,34 @@
 package e2e
 
 import (
-	"fmt"
-	"io"
-	"net/http"
-	"strings"
 	"time"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
-
 	"github.com/osmosis-labs/osmosis/v7/tests/e2e/chain"
-	"github.com/osmosis-labs/osmosis/v7/tests/e2e/util"
 )
 
-func (s *IntegrationTestSuite) Test001IBCTokenTransfer() {
-	var ibcStakeDenom string
-
+func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
```

Comment:
Why do we not care about the order anymore?

Reply:
The previous implementation checked for Len==3, which is what we used as just a proof of concept. This function now stores the coin struct of the wallet pre send and post send and then compares them to each other. The difference between the structs should be the amount that was sent. If not, something went wrong.

Reply:
Awesome, nice wok!